apps/business_central/exports/purchase_invoice/tasks.py

```
@@ -61,21 +63,34 @@ def __construct_purchase_invoice(self, body: PurchaseInvoice, lineitems: List[Pu
+        return purchase_invoice_payload, batch_purchase_invoice_lineitem_payload, dimensions
+
 
     def post(self, accounting_export, item, lineitem):
         '''
         Export the Journal Entry to Business Central.
         '''
 
-        purchase_invoice_payload, batch_purchase_invoice_payload = self.__construct_purchase_invoice(item, lineitem)
-        logger.info('WORKSPACE_ID: {0}, ACCOUNTING_EXPORT_ID: {1}, PURCHASE_INVOICE_PAYLOAD: {2}, BATCH_PURCHASE_INVOICE_PAYLOAD: {3}'.format(accounting_export.workspace_id, accounting_export.id, purchase_invoice_payload, batch_purchase_invoice_payload))
+        purchase_invoice_payload, batch_purchase_invoice_payload, dimensions = self.__construct_purchase_invoice(item, lineitem)
+        logger.info('WORKSPACE_ID: {0}, ACCOUNTING_EXPORT_ID: {1}, PURCHASE_INVOICE_PAYLOAD: {2}, BATCH_PURCHASE_INVOICE_PAYLOAD: {3},  DIMENSION_SET_LINE_PAYLOADS: {4}'.format(accounting_export.workspace_id, accounting_export.id, purchase_invoice_payload, batch_purchase_invoice_payload, dimension_set_line_payloads))
```

Ruff

Undefined name `dimension_set_line_payloads`

(F821)